docs/core/tools/custom-templates.md

```diff
@@ -7,11 +7,11 @@ ms.date: 05/20/2020
 
 # Custom templates for dotnet new
 
-The [.NET SDK](https://dotnet.microsoft.com/download) comes with many templates already installed and ready for you to use. The [`dotnet new` command](dotnet-new.md) isn't only the way to use a template, but also how to install and uninstall templates. You can create your own custom templates for any type of project, such as an app, service, tool, or class library. You can even create a template that outputs one or more independent files, such as a configuration file.
+The [.NET SDK](https://dotnet.microsoft.com/download) comes with many templates already installed and ready for you to use. The ["dotnet new" command](dotnet-new.md) isn't only the way to use a template, but also how to install and uninstall templates. You can create your own custom templates for any type of project, such as an app, service, tool, or class library. You can even create a template that outputs one or more independent files, such as a configuration file.
```

Comment:
I don't believe quotes prevent localization - presumably "new" here could be localized. It's safer to keep the code style.